lib/utapi/utapiReindex.js

```diff
@@ -1,5 +1,6 @@
 const UtapiReindex = require('utapi').UtapiReindex;
 const { config } = require('../Config');
 
-const reindex = new UtapiReindex(config.utapi && config.utapi.reindex);
+const reindexConfig = config.utapi.reindex ? config.utapi.reindex : config.utapi
```

Comment:
Is config.utapi always defined?

Reply:
yes - if for some reason it isn't, the UtapiReindex has default values to use